cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -409,6 +413,79 @@ func (s *Supervisor) startOpAMP() error {
 	return nil
 }
 
+func (s *Supervisor) stopOpAMP() error {
```

Comment:
Should we use this method in `Supervisor.Shutdown`?

Reply:
Updated